gui/sliders.py

```diff
@@ -0,0 +1,69 @@
+"""
+Main module for the application.
+
+This module initializes the main application window, sets up the canvas,
+adds sliders for various parameters, and binds events for canvas zooming.
```

Comment:
That's wrong description! I must have mistaken the modules. It is a description same as for the mainwindow. Will fix before the merge.